ggml-cuda.cu

```diff
@@ -84,6 +84,9 @@
 #include <cuda_runtime.h>
 #include <cublas_v2.h>
 #include <cuda_fp16.h>
+#define CUBLAS_TF32_TENSOR_OP_MATH CUBLAS_TENSOR_OP_MATH
+#define CUBLAS_COMPUTE_16F CUDA_R_16F
+#define CUBLAS_COMPUTE_32F CUDA_R_32F
```

Comment:
Wrap these in `#ifndef`.

Reply:
I know, but I don't know the version macro definition of the CUDA library

Reply:
You don't need it, do you?
Just check whether those macros are already defined and define them yourself if they are not.